src/components/ui/Button/LinkButton.tsx

```diff
@@ -21,13 +21,21 @@ function LinkButton({
   className = '',
   ...otherProps
 }: Props) {
+  const linkRef = useRef<HTMLAnchorElement | null>(null)
+
   return (
     <UILink
+      ref={linkRef}
       data-store-button
       className={`link-button ${className}`}
       data-button-variant={variant}
       data-button-inverse={inverse}
       data-button-disabled={disabled}
+      onFocus={(e: { preventDefault: () => void }) => {
+        e.preventDefault()
+
+        disabled && linkRef.current?.blur()
+      }}
```

Comment:
What is the intention here?  kind of prevent the focus and force the blur when the link is disabled?

```suggestion
      onFocus={(e: FocusEvent) => {
        e.preventDefault()

        disabled && linkRef.current?.blur()
      }}
```

We will need also
`import type { FocusEvent } from 'react'`

Reply:
hmm, it's because we can't have `pointer-events: none` with `cursor: not-allowed`